src/github.com/matrix-org/dendrite/syncserver/storage/output_room_events_table.go

```diff
@@ -232,7 +248,14 @@ func (s *outputRoomEventsStatements) Events(txn *sql.Tx, eventIDs []string) ([]g
 	return result, nil
 }
 
-func rowsToEvents(rows *sql.Rows) ([]gomatrixserverlib.Event, error) {
+type rowOrdering uint8
+
+const (
+	rowOrderForward = rowOrdering(iota)
+	rowOrderReverse
+)
```

Comment:
I'm not sure this is a good idea.
It might be clearer to just add the correct order by to the selects in the first place?